changelog.txt

```diff
@@ -1,5 +1,13 @@
 *** WooCommerce Payments Changelog ***
 
+= 1.2.0 - 2020-06-xx =
+* Add - 3DS support when using the pay for order page
+* Add - Display confirmation dialog when enabling manual captures
+* Add - Update the order when an authorised payment has expired
+* Add - Timeline view in transactions detail
+* Add - Tracks events for basic user actions
```

Comment:
Just a small tweak that I think we could use here for clarity, since `Tracks` isn't really known by users and merchants.

How about `Add tracking for basic user action events`?